Examples/MagicWeatherSwiftUI/Magic Weather SwiftUI.xcodeproj/project.pbxproj

```diff
@@ -19,7 +19,7 @@
 		559779AC25B72D3200C33DFB /* Constants.swift in Sources */ = {isa = PBXBuildFile; fileRef = 559779AB25B72D3200C33DFB /* Constants.swift */; };
 		559779B925B72DCF00C33DFB /* SampleData.swift in Sources */ = {isa = PBXBuildFile; fileRef = 559779B325B72DC000C33DFB /* SampleData.swift */; };
 		559779C025B72EE100C33DFB /* PurchasesDelegateHandler.swift in Sources */ = {isa = PBXBuildFile; fileRef = 559779BF25B72EE100C33DFB /* PurchasesDelegateHandler.swift */; };
-		B3D4400727220C6800A665B1 /* RevenueCat in Frameworks */ = {isa = PBXBuildFile; productRef = B3D4400627220C6800A665B1 /* RevenueCat */; };
+		5791CEC42742EF2400E50C4B /* RevenueCat.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 5791CEC32742EF2400E50C4B /* RevenueCat.framework */; };
```

Comment:
changes in this file should be from @NachoSoto's branch to add Catalyst support and use the framework directly instead of through SPM